pkg/app/app.go

```diff
@@ -65,6 +65,7 @@ func New() EnvdApp {
 		CommandResume,
 		CommandUp,
 		CommandVersion,
+		CommandInit,
```

Comment:
Please place the command before `CommandPause`. We follow the alphabetical order